index.html

```diff
@@ -174,16 +175,20 @@
 
         if (currentTime.getSeconds() === 0) {
             switch(currentTime.getMinutes()) {
-                case 0: chime0q.play(); chime0q.addEventListener('ended', chimeHourly); break;
                 case 15: chime1q.play(); break;
                 case 30: chime2q.play(); break;
                 case 45: chime3q.play(); break;
+            } else if (currentTime.getSeconds() === 38){
+              switch(currentTime.getMinutes()) {
+                case 59:
+                  chime0q.play();
+                  chime0q.addEventListener('ended', chimeHourly);
+              }
```

Comment:
The first part of the solution I'm suggesting: when the clock is on 38 seconds, verify if that minute is 59 (right before the full hour). If true, begin the full-hour-chimes.

Reply:
I'm using this order of rules because the promises of JS...